### PR TITLE
W7 router

### PR DESCRIPTION
**Pull request by karinasanchez04**
Implemented the router functionality.

### CODE REVIEW

File: source/scripts/script.js [hunk 1]
```diff
@@ -124,23 +137,28 @@ function createRecipeCards() {
 
 function bindRecipeCard(recipeCard, pageName) {
     recipeCard.addEventListener('click', e => {
-        if (e.path[0].nodeName == "A") return;
-        router.navigate(pageName);
+      if (e.path[0].nodeName == 'A') return;
```
Comment (kjl001) [inline source/scripts/script.js:140]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group23/cse110-fa21-group23/pullRequest?prid=8436588)
```suggestion
        if (e.path[0].nodeName == "A") return;
```

Comment (kjl001) [inline source/scripts/script.js:140]: Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/cse110-fa21-group23/cse110-fa21-group23/pullRequest?prid=8436588)
```suggestion
        if (e.path[0].nodeName == 'A') {return;}
```

File: source/scripts/script.js [hunk 2]
```diff
@@ -39,6 +44,7 @@ async function init() {
         if (event.key === 'Enter') {
             let searchSuccessful = await search();
             if (searchSuccessful) {
+                router.navigate(document.getElementById('search-query').value, false);
```
Comment (kjl001) [inline source/scripts/script.js:47]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group23/cse110-fa21-group23/pullRequest?prid=8436588)
```suggestion
                router.navigate(document.getElementById("search-query").value, false);
```

**Comment by nhilikeknee** [comment]
Great job with the router, @karinasanchez04. There is just one minor bug with going from a recipe page to the Settings page (the recipe page content still displays) so just take a look and fix that for your next pull request. I'm still going to merge this PR tho :) Closes #35

**Pull request merged**